Source/WebCore/platform/graphics/gstreamer/WebKitWebSourceGStreamer.cpp

```diff
@@ -880,14 +880,12 @@ static GstURIType webKitWebSrcUriGetType(GType)
 
 const gchar* const* webKitWebSrcGetProtocols(GType)
 {
-    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GLib port
-    static const char* protocols[4];
+    static std::array<const char*, 4> protocols;
     protocols[0] = "http";
     protocols[1] = "https";
     protocols[2] = "blob";
     protocols[3] = nullptr;
```

Comment:
Any reason to not use an initializer list here?

```suggestion
    static std::array<const char*, 4> protocols { "http", "https", "blob" };
```

Note that you do not need to pass `nullptr` at the end, the implementation of `std::array` takes care of using the default initializer for the array positions left after consuming the initializer list.